wp-includes/rest-api/rest-functions.php

```diff
@@ -226,17 +226,15 @@ function rest_get_url_prefix() {
  *
  * @since 4.4.0
  *
- * @todo Check if this is even necessary
- *
- * @param int    $blog_id Optional. Blog ID. Default of null returns URL for current blog.
- * @param string $path    Optional. REST route. Default '/'.
+ * @param string $namespace Plugin or theme namespace for the route.
+ * @param string $route    Optional. REST route. Default '/'.
  * @param string $scheme  Optional. Sanitization scheme. Default 'json'.
+ * @param int    $blog_id Optional. Blog ID. Default of null returns URL for current blog.
  * @return string Full URL to the endpoint.
  */
-function get_rest_url( $blog_id = null, $path = '/', $scheme = 'json' ) {
-	if ( empty( $path ) ) {
-		$path = '/';
-	}
+function get_rest_url( $namespace, $route = '/', $scheme = 'json', $blog_id = null ) {
```

Comment:
Oh, something else I thought about. With this pull request, there's no way to `get_rest_url()` for the API index.

Given the breaking change and the fact that you can't `get_rest_url()` for the API index, I think I'm actually :-1: for this pull request.